src/PrestaShopBundle/Form/Admin/Sell/Customer/GroupType.php

```diff
@@ -53,10 +53,8 @@ public function getParent(): string
     public function configureOptions(OptionsResolver $resolver): void
     {
         $resolver->setDefaults([
-            'placeholder' => null,
             'choices' => $this->groupByIdChoiceProvider->getChoices(),
             'choice_translation_domain' => false,
-            'autocomplete' => true,
```

Comment:
@M0rgan01 you sure it is required to fix the issue? It adds select2 to the input if I recall correctly.

Reply:
Yes, It was not present before the regression, and the rendering is not good with

![Capture d’écran du 2023-05-16 15-23-50](https://github.com/PrestaShop/PrestaShop/assets/36233446/9d505894-db51-49b4-ba3f-bc0266d842fb)